canvas_modules/common-canvas/src/common-properties/controls/expression/expression-builder/expression-builder.jsx

```diff
@@ -38,7 +38,7 @@ export default class ExpressionBuilder extends React.Component {
 		const value = (typeof newValue === "string") ? newValue : newValue.toString();
 		const somethingSelected = this.getCodemirrorState()?.selection.ranges.some((r) => !r.empty);
 		let cursor = this.getCodemirrorState()?.selection.main.head;
-		if (cursor === 0 && !somethingSelected) { // TODO: Doesn't work when I explicitly set the cursor to 0
+		if (isNaN(cursor) && !somethingSelected) { // TODO: Doesn't work when I explicitly set the cursor to 0
```

Comment:
Should the `TODO` be removed?

Reply:
I'll look into it

Reply:
If I switch the condition to `(isNan(cursor) || cursor === 0)` , the bug is still present. I'm not to sure about the comment, I can remove if need be.

Reply:
we should remove it since I don't know why it would be needed anymore.